lib/combo/names.ts

```diff
@@ -3,20 +3,21 @@ const ITEM_NAMES: {[k: string]: string} = {
   MM_ARROW_ICE: "Ice Arrows (MM)",
   MM_ARROW_LIGHT: "Light Arrows (MM)",
   MM_ARROWS_30: "30 Arrows (MM)",
-  MM_BOMBCHU: "1 Bombchu (MM)",
-  MM_BOMBCHU_10: "10 Bombchu (MM)",
-  MM_BOMBCHU_5: "5 Bombchu (MM)",
-  MM_BOMBER_NOTEBOOK: "Bomber's Notebook",
   MM_BOMB_BAG: "Bomb Bag (MM)",
+  MM_BOMBCHU: "1 Bombchu (MM)",
+  MM_BOMBCHU_10: "10 Bombchus (MM)",
```

Comment:
The plural of `Bombchu` is just `Bombchu` (no `s`) according to in-game text.

Reply:
Thanks for pointing that out, I see now that "Bombchus" as a plural was only introduced in more recent games. Back to Bombchu it is!